scripts/package.lua

```diff
@@ -40,7 +42,7 @@
 -- Make sure I've got what I've need to be happy.
 --
 
-	local required = { "git", "make", "gcc", "premake5", "zip" }
+	local required = { "git", "make", "gcc", "zip" }
```

Comment:
Is `premake5` on the path? It is calling through `os.execute()` which is spinning up a new shell environment. That's the only thing that immediately comes to mind, but you probably already considered that.